### ISSUE TITLE
[BUG] Raz's Pauldrons socket bonus value does not match its description

### ISSUE THREAD

**Issue by raethkcj** [JamminL/cata-classic-bugs#1410]
#### 1.1. Description
  The socket bonus for [Raz's Pauldrons](https://www.wowhead.com/cata/item=56318/razs-pauldrons) uses SpellItemEnchantment [4141](https://wago.tools/db2/SpellItemEnchantment?build=4.4.0.55056&filter[ID]=4141&page=1).
  
  This enchantment's text is "+10 Hit Rating", but its `EffectPointsMin_0` and `EffectPointsMax_0` columns are set to 20. This means it provides 20 hit rating when the gem requirements are met, not 10.
  #### 1.2. How to Reproduce
           1. Obtain Raz's Pauldrons
           2. Note your current hit rating
           3. Socket a red gem with zero hit into the pauldrons
           4. Observe that you gain 20 hit rating
  #### 1.3  Source Material
https://wago.tools/db2/SpellItemEnchantment?build=4.4.0.55056&filter[ID]=4141&page=1
#### 2.1. Description
  The text should match the effect; for an item of this level, it's likely the text is correct and the effect's value is incorrect.

**Comment by CharWS** [comment]
![raz](https://github.com/JamminL/cata-classic-bugs/assets/104176019/909a353c-5c72-48a2-bc9c-bfadc9181256)
it is notably how the item worked originally as well

**Comment by raethkcj** [comment]
Yeah, it'd be fair to leave the effect as-is and fix the tooltip string instead! It's not terribly overpowered, just confusing.

**Comment by raethkcj** [comment]
Some other enchants whose strings do not match the stats they provide. In this case due to `PointsMin` and `PointsMax` having inequal values.

https://wago.tools/db2/SpellItemEnchantment?build=4.4.2.58998&filter[ID]=2991%7C3820&page=1

E.g. [Arcanum of Burning Mysteries](https://www.wowhead.com/cata/spell=59970/arcanum-of-burning-mysteries) provides 30 intellect rather than the stated 26.
